pysph/sph/wc/shift.py

```diff
@@ -0,0 +1,226 @@
+from math import sqrt
+from compyle.api import declare
+from pysph.sph.equation import Equation
+from pysph.base.reduce_array import serial_reduce_array
+from pysph.solver.tools import Tool
+
+
+class SimpleShift(Equation):
+    """See the paper by R. Xu et al. (JCP 2009), equation(35)
+    """
+    def __init__(self, dest, sources, const=0.04):
+        self.beta = const
+        super(SimpleShift, self).__init__(dest, sources)
+
+    def py_initialize(self, dst, t, dt):
+        from math import sqrt
```

Comment:
Is this import needed? and shouldn't it be from `numpy import sqrt`?

Reply:
this is not needed! I tried to not import numpy at all, but now changed it to a numpy.sqrt operation as it is much clear.